plugins/polio/budget/api.py

```diff
@@ -52,50 +50,39 @@ class BudgetCampaignViewSet(ModelViewSet, CSVExportMixin):
         CustomFilterBackend,
     ]
 
+    def get_serializer_class(self):
+        if self.action == "create":
+            return BudgetProcessWriteSerializer
```

Comment:
Just curious, is this a common thing to do? Having a different serializer for write operations?

Reply:
I don't really know if it's a common thing to do, but sometimes it makes things clearer IMO.